docs/applications/setup.md

```diff
@@ -0,0 +1,20 @@
+# Setup
+
+This application provides a terminal based GUI for checking the validity of service configuration settings.
+
+## Message Flow
+
+N/A.
+
+## YAML Configuration
+
+N/A.
+
+## CLI Options
+
+In addition to any common options:
+
+```console
```

Comment:
`console`? Not `setup`?

Reply:
This is a code block, not a heading?

Reply:
Ah, that makes sense - I think the previous ones I saw were using `bash` not `console`